### ISSUE TITLE
video player: vertical video recorded on the phone camera have wrong size

### ISSUE THREAD

**Issue by pepegich** [flutter/flutter#24748]
## Steps to Reproduce
1. create flutter app
2. .
include dependences:
  image_picker: ^0.4.5
  video_player: 0.7.2
3. copy https://gist.github.com/pepegich/38d49ad04ec84d4e8b5da46d8477ddb1 code to main.dart
4. run on device
5. record vertical video by camera
6. record horizontalvideo by camera

result:
vertical video have same size as horizontal, but it is reproducess only video from camera
vertical video has size: 1920x1080 it is wrong
horizontal video has size: 1920x1080

as example, I add vertical and horizontal video from network, size of these is correct.

reproduced on devices:
google pixel 2
oneplus 5
nokia 6,1

```
[√] Flutter (Channel dev, v0.11.6, on Microsoft Windows [Version 10.0.17134.407], locale ru-RU)
    • Flutter version 0.11.6 at E:\Programming\git\flutter
    • Framework revision d44aa57c12 (12 days ago), 2018-11-14 11:09:56 -0800
    • Engine revision 114d33d3ca
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[√] Android toolchain - develop for Android devices (Android SDK 27.0.3)
    • Android SDK at C:\Users\admin\AppData\Local\Android\sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-P, build-tools 27.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[√] Android Studio (version 3.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 30.0.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[√] IntelliJ IDEA Community Edition (version 2018.2)
    • IntelliJ at C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2018.2.6
    • Flutter plugin version 27.1.3
    • Dart plugin version 182.3911.37

[!] Connected device
    ! No devices available
```

<details><summary>screenshot</summary><p>

![screenshot_20181126-211940](https://user-images.githubusercontent.com/11444783/49031692-21567b80-f1c4-11e8-9810-3c3b6ba540d6.png)

![screenshot_20181126-212005](https://user-images.githubusercontent.com/11444783/49031711-2ddad400-f1c4-11e8-9e69-39fbfe847a76.png)

![screenshot_20181126-212019](https://user-images.githubusercontent.com/11444783/49031715-2fa49780-f1c4-11e8-900b-6ded6da6045a.png)

![screenshot_20181126-212218](https://user-images.githubusercontent.com/11444783/49031721-33381e80-f1c4-11e8-8b1a-fce755807f50.png)

</p></details>

**Comment by franzejr** [comment]
I have this same issue!

**Comment by pepegich** [comment]
it is fixed in video_player: ^0.10.0+2

<details><summary>screenshot</summary><p>

![h](https://user-images.githubusercontent.com/11444783/52780465-a0d94580-3063-11e9-8f79-081b931aaf8a.jpg)
![v](https://user-images.githubusercontent.com/11444783/52780469-a3d43600-3063-11e9-83eb-6e24a10a5e7f.jpg)
</p></details>
